de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/49-de.metas.invoice/5448170_sys_FRESH-506_Invoice_UpdatePaymentRule_Process_Schedule.sql

```diff
@@ -0,0 +1,114 @@
+-- 12.07.2016 15:25
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process (AccessLevel,AD_Client_ID,AD_Org_ID,AD_Process_ID,AllowProcessReRun,Classname,CopyFromProcess,Created,CreatedBy,EntityType,IsActive,IsApplySecuritySettings,IsBetaFunctionality,IsDirectPrint,IsOneInstanceOnly,IsReport,IsServerProcess,IsUseBPartnerLanguage,LockWaitTimeout,Name,RefreshAllAfterExecution,ShowHelp,Statistic_Count,Statistic_Seconds,Type,Updated,UpdatedBy,Value) VALUES ('1',0,0,540700,'Y','de.metas.adempiere.process.ExecuteUpdateSQL','N',TO_TIMESTAMP('2016-07-12 15:25:44','YYYY-MM-DD HH24:MI:SS'),100,'U','Y','N','N','N','N','N','N','Y',0,'test','N','Y',0,0,'SQL',TO_TIMESTAMP('2016-07-12 15:25:44','YYYY-MM-DD HH24:MI:SS'),100,'testrux')
```

Comment:
wrong entitytype 'U', I might be wrong but I think it's not updated further down